client/src/btc/handlers/inbound.rs

```diff
@@ -151,18 +152,24 @@ impl<T: JsonRpcClient + 'static> InboundHandler<T> {
 }
 
 #[async_trait::async_trait]
-impl<T: JsonRpcClient + 'static> TxRequester<T> for InboundHandler<T> {
+impl<T: JsonRpcClient + 'static, S: Signer<CustomConfig>> TxRequester<T, S>
+	for InboundHandler<T, S>
+{
 	fn tx_request_sender(&self) -> Arc<TxRequestSender> {
 		self.tx_request_sender.clone()
 	}
 
-	fn bfc_client(&self) -> Arc<EthClient<T>> {
+	fn bfc_client(&self) -> Arc<EthClient<T, S>> {
 		self.bfc_client.clone()
 	}
 }
 
 #[async_trait::async_trait]
-impl<T: JsonRpcClient + 'static> Handler for InboundHandler<T> {
+impl<T: JsonRpcClient + 'static, S: Signer<CustomConfig>> Handler for InboundHandler<T, S>
+where
+	S: Send,
+	S: Sync,
```

Comment:
```suggestion
    T: JsonRpcClient,
	S: Signer<CustomConfig> + Send + Sync,
```
where 절에 다 넣든, impl에 넣든 하나로 통일하시죠

Reply:
다른 곳들도 마찬가지

Reply:
https://github.com/bifrost-platform/bifrost-relayer.rs/pull/151/commits/8d35d44a8091404098f34b690b6cc13854773175